### PR TITLE
Create migration to set progression_name based on activity section name

### PR DESCRIPTION
**Pull request by dmcavoy**
Creates a migration to update the progression_name to be the activity section name for activity sections that have at least 1 level and are in migrated scripts (21-22 curriculum).

Do I need to do something re-serialize everything once this has been updated?

## Testing story

Ran locally and checked that activity sections were updated appropriately.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — COMMENTED** [review]
this looks like the right idea, but I am wondering how it is going to play out on the levelbuilder machine and with script_json files. When this runs on the levelbuilder machine, it will update the database, but not the .script_json files. if no other steps are taken, then the levelbuilder machine will seed from the unmodified .script_json files and wipe out these changes. does that sound accurate? 

I guess one alternative would be to just run this migration once locally, regenerate the .script_json files, and then commit the result. this would get the results to all environments.

there could be other options too. sorry for missing this earlier.

File: dashboard/db/migrate/20210119225102_copy_activity_section_title_to_progression_name.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            section.progression_name = section.name
+            section.save!
+            section.script_levels.each do |sl|
+              sl.progression = section.name
+              sl.save!
+            end
+          end
+        end
+      end
+    end
```
Comment (davidsbailey) [inline dashboard/db/migrate/20210119225102_copy_activity_section_title_to_progression_name.rb]: another idea would be to take all the code inside this migration and move it to a oneoff script, and then add a line here which rewrites the .script_json before moving onto the next script. then you could just run this on your local machine (or levelbuilder) to get the change you want into the .script_json files.

**Comment by dmcavoy** [comment]
@davidsbailey I have update the PR to have the script_json files with the progression_name updates instead of the migration

**Review by bethanyaconnor — APPROVED** [review]
This looks good to me! It might be worth mentioning this PR to the dotd after merging as it could cause merge conflicts with the scoop.

**Review by davidsbailey — APPROVED** [review]

**Pull request merged**